src/overlays/actors/ovl_Bg_Menkuri_Eye/z_bg_menkuri_eye.h

```diff
@@ -8,9 +8,9 @@ struct BgMenkuriEye;
 
 typedef struct BgMenkuriEye {
     /* 0x0000 */ Actor actor;
-    /* 0x014C */ char unk_14C[0x4];
-    /* 0x0150 */ Collider collider;
-    /* 0x0168 */ char unk_168[0x48];
+    /* 0x014C */ s16 unk_14C;
+    /* 0x0150 */ ColliderJntSph collider;
+    /* 0x0170 */ ColliderJntSphItem colliderItem;
```

Comment:
May want to make `colliderItem` an array of length 1 (and rename it to `colliderItems`), even with length 1 it's handled as an array (if you do that the `&` prefixing `colliderItem` in `Collider_SetJntSph` in Init can be removed)